makefile

```diff
@@ -16,14 +15,11 @@ else
 endif
 	mv chrome.crx chrome-browserpass.crx
 
-.PHONY: firefox
-firefox:
-	cp chrome/{*.html,*.css,*.js,*.png,*.svg} firefox/
-
 .PHONY: js
 js: deps $(JS_OUTPUT)
```

Comment:
If `$(JS_OUTPUT)` files have been build but `{chrome,firefox}-host.json` and/or `firefox/{*.html,*.css,*.js,*.png,*.svg}` have not, this target will exit with `Nothing to do for "js"`.

Reply:
Or, im wrong because js is a phony target. In that case: sorry for the noise

Reply:
Tested this, to be sure:

```
$ make js
$ rm chrome-host.json
$ make js
```

The `chrome-host.json` reappeared, so seems to be fine 👍